machinelearning/intel-tensorflow/Singularity.mkl.def

```diff
@@ -0,0 +1,138 @@
+Bootstrap: debootstrap
+OSVersion: bionic
+MirrorURL: http://us.archive.ubuntu.com/ubuntu/
+
+%help
+    This Singularity definition contains Intel(R) Optimizations for
+    TensorFlow.
+
+    Download whl from the following URL and save in the same directory as this file.
+    https://pypi.org/project/intel-tensorflow/1.12.0/#files
+
+    NOTE: Please update TF_WHL with wheel name, otherwise build will fail.
+
+    Use below commands:
+
+    *Build singularity container* - This will build an immutable container (recommended in production environments):
+    sudo singularity build <mysingularity_name>.sif Singularity.mkl.def
+
+    *To build development container* - Where you can install additional components as needed:
+    sudo singularity build --sandbox <mysingularity_name>/ Singularity.mkl.def
+
+    *Shell into singularity container*
+    sudo singularity shell --contain --writable <mysingularity_name>/
+
+    --writable option must be provided to install any packages.
+    You cannot use --writable option with immutable containers
+
+    *Exec into singularity container*- To run any command from host inside container
+    sudo singularity exec --contain <mysingularity_name>/ env
+
+    *Run singularity container in background and access notebook* - This will run %starscript section commands
+    sudo singularity instance start <mysingularity_name>/ demotest
+
+    *Check running instances* - This should list demotest instance started above
+    sudo singularity instance list
+
+    *Get the token to access notebook locally* - copy the listed token
+    sudo singularity exec instance://demotest jupyter notebook list
+
+    If you are running singularity instance in remote machine.
+    Execute below command to establish tunnel to forward traffic from remote machine to localhost
+    Ex: ssh -L 8888:localhost:8888 <remote_machine_name>
+
+    Access the given url from your localhost and enter the access token
+    Ex: localhost:8888
+
+%setup
+    mkdir -p ${SINGULARITY_ROOTFS}/root/.jupyter/
+    cp jupyter_notebook_config.py ${SINGULARITY_ROOTFS}/root/.jupyter/
+
+%files
+    # Download whl from the following URL and save in the same directory as this file.
+    # https://pypi.org/project/intel-tensorflow/1.12.0/#files
+    # Uncomment the following line before building the container. Please use different whl file name as needed.
+    # intel_tensorflow-1.12.0-cp27-cp27mu-manylinux1_x86_64.whl /
+
+    # Copy sample notebooks.
+    notebooks /notebooks
+
+    # Jupyter has issues with being run directly:
+    #   https://github.com/ipython/ipython/issues/7062
+    # We just add a little wrapper script.
+    run_jupyter.sh /
+
+%post -c /bin/bash
+
+    # Uncomment below line and update previously downloaded tensorflow wheel name(if different)
+    # TF_WHL="intel_tensorflow-1.12.0-cp27-cp27mu-manylinux1_x86_64.whl"
```

Comment:
Uncomment this line as well. We'll make an assumption here that the `intel_tensorflow-1.12.0-cp27-cp27mu-manylinux1_x86_64.whl` is what will be downloaded. Leave these uncommented, since it's needed for the build script to work.

If the user changes which one they grab, then they can edit it.